config/secrets_example.yml

```diff
@@ -14,11 +14,23 @@ development:
   secret_key_base: secret
   admin_user: dev
   admin_password: devpwd
+  newrelic:
+    app_name: i14y (PUT YOUR NAME HERE) (<%= Rails.env %>)
+    enabled: false
+    host: gov-collector.newrelic.com
+    license_key: newreliclicensekeygoeshere
+    log_level: info
 
 test:
   secret_key_base: secret
   admin_user: test
   admin_password: testpwd
+  newrelic:
+    app_name: i14y (PUT YOUR NAME HERE) (<%= Rails.env %>)
+    enabled: false
+    host: gov-collector.newrelic.com
+    license_key: newreliclicensekeygoeshere
```

Comment:
Do we need these keys if `enabled` is set to `false`?

Reply:
Yea, unfortunately, if we want to avoid this noised in local spec runs:

```
 ~/Documents/GitHub/i14y   SRCH-4362 ±  rspec spec                                      
** [NewRelic] FATAL : Failed ERB processing configuration file. This is typically caused by a Ruby error in <% %> templating blocks in your newrelic.yml file.
** [NewRelic] FATAL : undefined method `[]' for nil:NilClass
Run options: include {:focus=>true}
```